File: Code/js/quadratic_roots.js
a = prompt("Enter Coefficient a");
b = prompt("Enter Coefficient b");
c = prompt("Enter Coefficient c");

let d = b ** 2 - (4 * a * c);
if (d > 0)
{
  console.log((-b + Math.sqrt(d)) / (2 * a));
  console.log((-b - Math.sqrt(d)) / (2 * a));
}
else if (d == 0)
  console.log(-b / (2 * a));
else
  console.log("Complex Roots");

File: Code/py/quadratic_roots.py
from math import sqrt

a = float(input("Enter Coefficient a: "))
b = float(input("Enter Coefficient b: "))
c = float(input("Enter Coefficient c: "))

d = b ** 2 - (4 * a * c)
if d > 0:
    print((-b + sqrt(d)) / (2 * a))
    print((-b - sqrt(d)) / (2 * a))
elif d == 0:
    print(-b / (2 * a))
else:
    print("Complex Roots")